doc/examples/images/ex01.ps.dvc

```diff
@@ -0,0 +1,5 @@
+outs:
+- md5: 08a2e5d2a7f9135d82ea0ae2ac99579f
+  size: 338598
+  hash: md5
```

Comment:
@PaulWessel all the new `.dvc` files added by me have the `hash: md5` line, which is something new in newer dvc versions. Maybe it's easier if you can upgrade your dvc version.